pyproject.toml

```diff
@@ -2,6 +2,9 @@
 requires = ["uv_build >= 0.8.17, <0.9.0"]
 build-backend = "uv_build"
 
+[tool.uv.build-backend]
```

Comment:
@safoinme Can you check if this behaves correctly in the following cases:
- Fresh install in a virtualenv
- Upgrading an existing ZenML installation

Reply:
Thanks!